src/core/Config.h

```diff
@@ -86,15 +86,24 @@ enum UIScaleFilter {
 	UIFilterBilinear = 1
 };
 
+
+namespace {
+
+const uint32_t NUM_OF_KEYS_PER_ACTION = 2;
+const uint32_t DEFAULT_ACTION = 0;
```

Comment:
There is nothing "default" about the first action binding. Also introducing names and then renaming then in the same commit series is pointless.